ShoppingList/ShoppingList/View Controllers/AddTaskViewController.swift

```diff
@@ -0,0 +1,236 @@
+//
+//  AddTaskViewController.swift
+//  ShoppingList
+//
+//  Created by Jerrick Warren on 5/20/19.
+//  Copyright © 2019 Lambda School Labs. All rights reserved.
+//
+
+import UIKit
+
+class AddTaskViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UITextFieldDelegate {
+   
+
+    // Load sample values
+    
+    
+    
+    var sampleCategories = ["Kitchen", "Bathroom", "Dining Room", "Living Room" ,"Master Bedroom"]
+    var sampleTask = ["Clean up", "Wipe Down", "Wash dishes", "Vaccum", ]
+    
+    // Properties
+    var categories: [Category]?
+    var userController: UserController?
+    var currentUser: User?
+    var householdController: HouseholdController?
+    var taskController = TaskController()
+    var categoryController = CategoryController()
```

Comment:
shouldn't be initialized... initialized in TabBarVC

Reply:
Category isn't being passed from TabBarVC right now, but we need to refactor that. This is probably why it didn't work without initialization. @jWarrenDev